src/main/java/service/DataWriter.java

```diff
@@ -1,18 +1,8 @@
 package service;
 
-import java.io.*;
+public interface DataWriter {
 
-public class DataWriter {
+    void writeDataToFile(String fileName, String data);
 
-    public void writeDataToFile(String fileName, String data) {
-
-        try (FileWriter writer = new FileWriter(fileName, true);
-             BufferedWriter bufferedWriter = new BufferedWriter(writer);
-             PrintWriter printWriter = new PrintWriter(bufferedWriter)) {
-
-            printWriter.print(data);
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-    }
+    String getPropertyValue(String propertyName);
```

Comment:
the method "get..." slightly mismatched to the class name "...writer"

Reply:
done